### ISSUE TITLE
Documentation Amazon ECS using AWS Copilot

### ISSUE THREAD

**Issue by avelino** [prest/prest#640]
write documentation page on how to deploy prestd in [Amazon ECS using AWS Copilot](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/getting-started-aws-copilot-cli.html)

example: https://github.com/prest/prest/blob/main/docs/deployment/docker.en.md

**Comment by avelino** [comment]
@george-silva as you have experience can you help us by writing this page?

**Comment by george-silva** [comment]
Yes I can write it! I'll check where the docs are and I'll do a PR.
